python/lsst/ip/diffim/dcrModel.py

```diff
@@ -545,9 +486,52 @@ def calculateNoiseCutoff(self, maskedImage, statsCtrl, sigma, bufferSize,
         convergeMask = mask.getPlaneBitMask(convergenceMaskPlanes)
 
         backgroundPixels = mask[bboxShrink].array & (statsCtrl.getAndMask() | convergeMask) == 0
-        noiseCutoff = sigma*np.std(maskedImage[bboxShrink].image.array[backgroundPixels])
+        noiseCutoff = np.std(maskedImage[bboxShrink].image.array[backgroundPixels])
```

Comment:
This seems like it would me more natural (and probably faster) if you just used afwMath.makeStatistics setting the appropriate bit you want andMask | convergeMask in your own control object. This would save a few loops over the array.

Reply:
I agree it would be cleaner with all the bits set in the statsCtrl object, but I'm going to punt on this change for now. I started writing it, but then realized that I would have to pass around several additional statsCtrl objects, because sometimes I want to exclude pixels with either a set of bad masks or "convergence" masks set, and sometimes I want to exclude pixels with the set of bad masks but NOT the convergence masks set.